### ISSUE TITLE
unable to use LightGBMClassifier in R due to wrong datatype

### ISSUE THREAD

**Issue by elenaterenzi** [Azure/mmlspark#375]
I have an R data.frame object which I would like to pass to the ml_light_gbmclassifier function using the following code:
`model= ml_light_gbmclassifier(maxDepth=20, x=trainDF, featuresCol=feat, labelCol=lab,numIterations=10)`
it throws an error saying that it cannot convert object data.frame to Spark DataFrame.
I even tried using SparkR and creating a SparkDataFrame object but this is still not accepted, and I receive the following:

`Error : Unable to retrieve a Spark DataFrame from object of class SparkDataFrame`

**Comment by imatiach-msft** [comment]
@elenaterenzi I added an example to clear things up here:
https://github.com/Azure/mmlspark/pull/377
I also added the ability to create "unfit" or "untrained" models since that functionality was missing and wouldn't work with some estimators that took another estimator as a parameter (eg train regressor/train classifier).

**Comment by mhamilton723** [comment]
@imatiach-msft  can this issue be resolved?

**Comment by imatiach-msft** [comment]
@mhamilton723 @elenaterenzi  yes, we resolved over skype, I think we can close this now.  There are some issues with default params that need to be fixed in the R wrappers though that I need to look into more.

**Comment by ranadeepsingh** [comment]
We are confirming closure of this issue as it was resolved by PR #377, which added support for untrained models and provided updated usage examples for R in Sparklyr. This fix and documentation update have been included in published releases since v0.14.
